doc/source/f2py/usage.rst

```diff
@@ -79,8 +80,7 @@ distinguished by the usage of ``-c`` and ``-h`` switches:
      functions. This feature enables using arbitrary C functions
      (defined in ``<includefile>``) in F2PY generated wrappers.
 
-     This option is deprecated. Use ``usercode`` statement to specify
-     C code snippets directly in signature files
+     .. note:: This option is deprecated. Use ``usercode`` statement to specify C code snippets directly in signature files.
 
    ``--[no-]wrap-functions``
 
```

Comment:
I think the extra blank line here is causing the following text to be [formatted](https://11184-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/f2py/usage.html) with a quotation line

Reply:
Fixed.